.github/workflows/gh-actions.yml

```diff
@@ -291,15 +291,3 @@ jobs:
       - run: make -f ./targets/esp8266/Makefile.travis install-noapt
       - run: make -f ./targets/esp8266/Makefile.travis script
 
-  Notification:
-    runs-on: ubuntu-latest
-    if: github.event_name == 'push' && github.repository == 'jerryscript-project/jerryscript'
```

Comment:
How about?
```
if: false && github.event_name == 'push' && github.repository == 'jerryscript-project/jerryscript' # temporary disable this job
```

Reply:
Yeah, maybe this will be the best. Updated to use this format.